nilearn/_utils/docs.py

```diff
@@ -847,6 +847,22 @@ def custom_function(vertices):
         and the ``keep_masked_labels`` parameter will be removed in 0.15.
 """
 
+# keep_masked_maps
+docdict["keep_masked_maps"] = """
+keep_masked_maps : :obj:`bool`, optional
+    If True, masked atlas with invalid maps (maps that contain only
+    zeros after applying the mask) will be retained in the output, resulting
+    in corresponding time series containing zeros only. If False, the
+    invalid maps will be removed from the trimmed atlas, resulting in
+    no empty time series in the output.
+
+    .. deprecated:: 0.10.2.dev
+
+        The 'True' option for ``keep_masked_maps`` is deprecated.
+        The default value will change to 'False' in 0.13,
+        and the ``keep_masked_maps`` parameter will be removed in 0.15.
+"""
```

Comment:
@mtorabi59 
entries in this docs.py are supposed to be sorted alphabetically so this should probably be moved.

I can do it in #3621 after this is merged, just wanted to let you know.  😉

Reply:
@Remi-Gau I'll have that in mind for next time! Thnx!

Reply:
no worries but having things sorted can become helpful for long lists like this (also helps make sure you prevent the blood pressure of maintainers with sub-clinical  OCD from rising too much)